modules/ROOT/pages/_partials/include-insights-charts.adoc

```diff
@@ -0,0 +1,385 @@
+ 
+//end::app-detail-requests[]
+//-----------------------//
+//
+//-----------------------//
+//  APP DETAIL ERRORS: Errors tab
+//tag::app-detail-errors[]
+Failed Outbound Request Volume::
+Total number of failed outbound requests at specific time points
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Top 20 Failed Requests by Total Count::
+Total number of inbound or outbound requests to the 20 endpoints with the most failed requests
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Top 20 Failed Outbound Requests by Total Count::
+Total number of unsuccessful outbound requests to the 20 endpoints with the most failed requests
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+//end::app-detail-errors[]
+//-----------------------//
+//
+//-----------------------//
+// APP DETAIL JVM: JVM tab
+//tag::app-detail-jvm[]
+//TODO: Old Gen GC Count -> Old Generation Garbage Collection Count (SATHISH)
+Old Gen GC Count::
+Total number of times old generation Java garbage collection occurred
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+//TODO: Old Gen GC Duration -> Old Generation Garbage Collection Duration (SATHISH)
+Old Gen GC Duration::
+Amount of time taken for old generation Java garbage collection
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+//TODO: Young Gen GC Count -> Young Generation Garbage Collection Count (SATHISH)
+Young Gen GC Count::
+Total number of times young generation Java garbage collection occurred
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+//TODO: Young Gen GC Duration -> Young Generation Garbage Collection Duration (SATHISH)
+Young Gen GC Duration::
+Amount of time taken for young generation Java garbage collection
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Heap Committed::
+Amount of memory the JVM guarantees to be available for the heap
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Heap Usage::
+Amount of physical memory used
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Thread Count::
+Number of active threads currently running in the JVM
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Committed Virtual Memory::
+Amount of virtual memory the operating systems guarantees to be available for the JVM
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+CPU Utilization::
+CPU work as a percentage of total CPU time
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+//end::app-detail-jvm[]
+//------------------------------------------------------//
+
+//========================//
+//API DETAILS: ALL METRICS//
+//========================//
+//
+// API DETAILS: Header-level metrics
+//-------------------------
+//tag::api-detail-header[]
+
+//TODO: Api -> API (SATHISH)
+Type::
+API implementation type (*API*)
```

Comment:
Just checking that API is bold because it's a UI element.

Reply:
Yes, that's what people see in the UI.